source/docs/articles/sites/code/unsupported-modules-plugins.md

```diff
@@ -6,56 +6,92 @@ keywords: modules, plugins, unsupported, drupal, wordpress
 This article lists modules and plugins that do not work with or are currently unsupported on the Pantheon platform.
 We do not prevent you from installing and using these plugins/modules; however, they will not work as expected and we cannot provide troubleshooting support.
 
-##Drupal Modules
-
-- APC
+## Drupal Modules
+<hr>
+### APC
+**Issue**: APC lacks the ability to span multiple server environments.
```

Comment:
We have APC cache built into the platform, so wondering if we should say that? @timani @newtoid  Can you look at this one?